docs/Custom-brimcap-load-Config.md

```diff
@@ -0,0 +1,433 @@
+further modify it to suit your needs.
+```
+    shaper: |
+      type alert = {
+        timestamp: time,
+        event_type: bstring,
+        src_ip: ip,
+        src_port: port=(uint16),
+        dest_ip: ip,
+        dest_port: port=(uint16),
+        vlan: [uint16],
+        proto: bstring,
+        app_proto: bstring,
+        alert: {
+          severity: uint16,
+          signature: bstring,
+          category: bstring,
+          action: bstring,
+          signature_id: uint64,
+          gid: uint64,
+          rev: uint64,
+          metadata: {
+            signature_severity: [bstring],
+            former_category: [bstring],
+            attack_target: [bstring],
+            deployment: [bstring],
+            affected_product: [bstring],
+            created_at: [bstring],
+            performance_impact: [bstring],
+            updated_at: [bstring],
+            malware_family: [bstring],
+            tag: [bstring]
+          }
+        },
+        flow_id: uint64,
+        pcap_cnt: uint64,
+        tx_id: uint64,
+        icmp_code: uint64,
+        icmp_type: uint64,
+        tunnel: {
+          src_ip: ip,
+          src_port: port=(uint16),
+          dest_ip: ip,
+          dest_port: port=(uint16),
+          proto: bstring,
+          depth: uint64
+        },
+        community_id: bstring
+      }
+      filter event_type=alert | put . = shape(alert) | rename ts=timestamp
+```
+
+A full description of all that's possible with shaping configurations is beyond
+the scope of this article. However, this configuration is quite simple and can
+be described in brief.
+
+1. The `type alert` defines the names, [data types](https://github.com/brimdata/zed/blob/main/docs/formats/zson.md#33-primitive-values),
+and hierarchical locations of expected fields in the input records. Here we've
+defined a single "wide" shape for _all_ alerts we've known Suricata to
+generate, which is convenient because it allows Brim to easily display them in
+columnar format.
+
+2. The `filter event_type=alert` trims the processed EVE events to only alerts.
+If you want to let through more Suricata data besides just alerts, you could
+remove this part of the pipeline. If so, you'll likely want to explore the
+additional data and create shapers to apply proper data types to them, since
+this will be a prerequisite for doing certain Zed queries with the data (e.g.
+a successful CIDR match requires IP addresses to be stored as `ip` type, not
+the string type in which they'd appear in unshaped JSON).
+
+3. The `put . = shape(alert)` applies the shape of the `alert` type to each
+input record. With what's shown here, additional fields that appear beyond
+those specified in the shaper (e.g. as the result of new Suricata features or
+your own customizations) will still be let through this pipeline and stored in
+the Zed Lake. If this is undesirable, add `| put . = crop(alert)` downstream of
+the first `put`, which will trim these additional fields.
```

Comment:
This is more CYA along the lines of the prior comment. `shape()` used to include `crop()`, but we took that out to err on the side of letting unexpected data through so the user would at least get to notice it. But since it would create problems with columnar layout in Brim, I wanted to make sure I at least inform them about `crop()` as an "a la carte" option so I can say I've at least warned them about it.